src/store/api/weatherProviders/accuweather.js

```diff
@@ -0,0 +1,100 @@
+import * as utils from './weatherutils';
+import getAPIData from '../apiutils';
+
+export const getCurrentPositionFromAPI = async state => {
```

Comment:
Passing entire `state` here is not a good idea. This function should be generic and should know nothing about internal structure of your store. The point here to make this api reusable so other parts of the app can consume it without any problems. The simplest case is to consider using api from component, store and outside of the framework.

In this case I'd make this fn accept `latitude` and `longitude`